analyses/fastqc-analysis/README.md

```diff
@@ -0,0 +1,52 @@
+# Pipeline for FastQC quality control tool for high throughput sequence data analysis
+
+## Usage
+
+`run-fastqc-analysis` is designed to be run as if it was called from this module directory even when called from outside of this directory.
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `run-fastqc-analysis.sh`: define path to input data with `*R2*.fastq.gz` files.
+
+### Run module on an interactive session on HPC
```

Comment:
Just to be extra clear, should the README include a sentence or something to specify that the bash script should not be run directly from the login node on HPC, but specifically from an interactive compute node?

Reply:
That is a great observation, thank you. I addressed this here 2ed530364a0df49469837ffb6e7eb41e0fff674c